README.md

```diff
@@ -21,6 +21,7 @@ Tools we are using:
 - **Fork this repo**
 - Clone repo locally (clone **your fork**, do NOT clone the main repo)
 - Configure a remote for the main repo (replace upstream with whatever naming makes sense to you) `git remote add upstream git@github.com:Drupal4Gov/Drupal-GovCon-2017.git`
+- First Run: from the repo root, go to /box/project.local.yml, anc change line 5 from `local: capitalcamp.local` to `local: self`
```

Comment:
- I think this should make a distinction between whether you intend to work from inside or outside the vm. We need to standardize on this as a team.
- `s/anc/and`